getx_demo/lib/main.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+
+import 'package:get/get_navigation/get_navigation.dart';
+
+import 'package:getx_demo/screens/loan_application_screen.dart';
+import 'package:getx_demo/screens/pay_screen.dart';
+import 'package:getx_demo/screens/top_up_screen.dart';
+import 'package:getx_demo/screens/transactions_screen.dart';
+
+void main() {
+  runApp(
+    GetMaterialApp(
+      routes: {
+        TransactionsScreen.route: (_) => const TransactionsScreen(),
```

Comment:
Try to use `get_cli` to make routes, and `modules` with theirs `controllers`, `views` and `bindings` folders.